idea/testData/resolve/references/InMethodVarargParameter.kt

```diff
@@ -0,0 +1,9 @@
+package test
+
+class A
+
+class Test {
+    fun some(vararg a: <caret>A) = 12
```

Comment:
Let also have a tests which checks that we can resolve to `a` param symbol from function body `fun some(vararg a A) = <caret>a` and from by-name parameter usage `Test().some(<caret> a = A())`

Reply:
For the second test case I assume, you meant `Test().some(<caret>a = arrayOf(A()))` right?